examples/competitor_researcher/setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Create a virtual environment
+echo "Creating virtual environment..."
+python3 -m venv ~/.venvs/competitor_researcher
+
+# Activate the virtual environment
+echo "Activating virtual environment..."
+source ~/.venvs/competitor_researcher/bin/activate
+
+# Install libraries from requirements.txt 
+echo "Installing libraries from requirements.txt..."
+pip install -r requirements.txt
+
+# Login to your account
+echo "Login to your Composio acount"
+composio composio login
```

Comment:
This line doesn't look correct. Have all this scripts been tested?

Reply:
I have changed, Verfied the working.